.github/workflows/tox.yml

```diff
@@ -12,18 +12,28 @@ on:
   schedule:
     - cron: '0 0 * * MON'
 
-defaults:
-  run:
-    shell: bash
-
 jobs:
   stable:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: ['ubuntu-latest', 'windows-latest']
+        os: ['ubuntu-latest']
         python-version: [3.6, 3.7, 3.8, 3.9, '3.10', 'pypy-3.7']
+        shell: ['bash']
+        include:
+          - os: 'windows-2019'
+            python-version: '3.10'
+            shell: 'bash'
+          - os: 'windows-2019'
+            python-version: '3.10'
+            shell: 'pwsh'
+          - os: 'windows-2019'
+            python-version: '3.10'
+            shell: 'cmd'
```

Comment:
Quick question: I don't have a particular preference, but what is the reason for using 'windows-2019' rather than 'windows-latest'?

Reply:
2022 that came out a couple days ago broke building wheels due to not finding the compiler or something. Your tests had run on 2019 and passing, so I reverted for now.

Long run it would be good to figure out what's going on there.

Reply:
Right, I remember having similar issues on my local machine. It's C++ Build Tools that you're talking about I presume. I know that's needed to build several packages in Python, so likely this issue will have been flagged in the general Python community already. Let's keep an eye on that.